Utils/old_content_branch.py

```diff
@@ -190,6 +206,21 @@ def edit_all_packs(new_to_version):
 parser.add_argument('-v', '--new-to-version', help='The new to version to assign.', required=True)
 
 
+def edit_reputations_json(new_to_version):
+    print("Updating reputations.json\n")
+    rep_json_path = "Packs/NonSupported/IndicatorTypes/reputations.json"
+    with open(rep_json_path, 'r') as f:
+        rep_content = ujson.load(f)
+
+    for reputation in rep_content.get('reputations', []):
+        if parse_version(reputation.get('toVersion', "99.99.99")) > parse_version(new_to_version) > \
+                parse_version(reputation.get("fromVersion", "0.0.0")):
+            reputation['toVersion'] = new_to_version
```

Comment:
are we adding the fromversion of this in the content creator? i don't think it's needed here @yaakovi WDYT?

Reply:
you can ignore the `fromversion`.
this can be relevant only for `toversion` lower than 4.0.0

Reply:
done